app-admin/prelude-manager/prelude-manager-3.0.0.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools eutils systemd
+
+DESCRIPTION="Bus communication for all Prelude modules"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/3.0.0/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="tcpwrapper xml geoip dbx systemd"
+
+RDEPEND="net-libs/gnutls
+	dev-libs/libprelude
+	dbx? ( dev-libs/libpreludedb )
+	tcpwrapper? ( sys-apps/tcp-wrappers )
+	xml? ( dev-libs/libxml2 )
+	geoip? ( dev-libs/libmaxminddb )"
+
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+PATCHES=(
+	"${FILESDIR}/${P}-conf.patch"
+	"${FILESDIR}/${P}-configure.patch"
+	"${FILESDIR}/${P}-run.patch"
+)
+
+src_prepare() {
+	default_src_prepare
+
+	mv "${WORKDIR}/${PF}/configure.in" "${WORKDIR}/${PF}/configure.ac" || die "mv failed"
+
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--localstatedir=/var \
+		$(use_enable dbx libpreludedb) \
+		$(use_with tcpwrapper libwrap) \
+		$(use_enable xml xmlmod) \
+		$(use_enable geoip libmaxminddb)
+}
+
+src_install() {
+	default_src_install
+
+	rm -rv "${D}/run/${PN}" || die "rm failed"
+
+	prune_libtool_files --modules
+
+	if use systemd ; then
+		systemd_dounit "${FILESDIR}/${PN}.service"
+		systemd_newtmpfilesd "${FILESDIR}/${PN}.run" "${PN}.conf"
+	else
+		newinitd "${FILESDIR}/${PN}.initd" "${PN}"
+	fi
+}
+
+pkg_postinst() {
+	if use systemd ; then
+		systemd-tmpfiles --create "${PN}.conf"
```

Comment:
I am not a systemd user but I suspect this is not required. None of the ebuilds in the tree that install a tmpfile does this. Likewise, the next line also appears to be not used by other packages. If we decide this to be unnecessary, then we can remove systemd use flag too since this package does not make use of any systemd-specific features.